projects/wallet/LeatherWallet-en.md

```diff
@@ -0,0 +1,69 @@
+
+## Project Name: LeatherWallet
+LeatherWallet
+
+
+## Category 
+Wallet, filecoin safe wallet
+<!--developer tooling, application, wallet, infrastructure, etc-->
+
+## Project Description
+Leather is a Filecoin Wallet application. It is an offline wallet, support complex operation of normal address type and bls address. You can download the source code and build from the code. Since it can be run on a totally offline computer, you can create and manage filecoin wallets in safety. 
+Different from other wallet, it supports all the complex operation of the owner wallet in filecoin, such as WithdrawBalance, ChangeOwnerAddress, ChangeWorkerAddress and so on.
+
+<!--Describe your project in a few sentences. -->
+
+## Previews
+This is the wallet homepage, showing the wallet list.
+![wallet list](https://user-images.githubusercontent.com/58079255/132000093-6e8c312e-dc12-46f5-a1f0-a7fdc1bad713.jpg)
+Create a new wallet
+![create a new wallet](https://user-images.githubusercontent.com/58079255/132000127-259c8d07-5285-478b-9801-7f5434d51d24.jpg)
+Choose an operation, supporting other operations besides transfer money.
+![choose an operation](https://user-images.githubusercontent.com/58079255/132000177-a40462cd-fb8b-4973-9e7d-72d32a30706e.jpg)
+Fill the information to transfer money
+![fill in the form](https://user-images.githubusercontent.com/58079255/132000190-55f73785-f8d4-4fc2-a7ec-7b54b16096a7.jpg)
+Show the sign message result
+![sign](https://user-images.githubusercontent.com/58079255/132000202-53aae829-7b0e-4e24-add4-bc8b98c45a8b.jpg)
+The final step is using a qr code scanner to get the sign string and send it out from our customized lotus, which is opensource too, to send the message into the message pool.
+
+<!--Add some screenshots to give a preview of your product-->
+
+## Target Audience
+Everybody can use it to create filecoin wallet and store the wallet safety. Miner owners can use the offline wallet to withdraw fil from miner account.
+<!--Describe who will be your project's users-->
+
+## Rough estimated user base (if applicable)
+about 5 miner shareholders
+<!--How many users do you have right now?-->
+
+## Github repo
+https://github.com/joyceqingling/lotus-offchain-wallet
+<!--Attach a link to your GitHub repo if it's OSS-->
+
+## Website
+https://github.com/joyceqingling/lotus-offchain-wallet
+<!--Link your website if available-->
+
+## Docs
+https://github.com/slate-engineering/wallet
+<!--Including a link to your project docs!-->
+
+## Team Info
+People at Protocol Labs build protocols, tools, and services to radically improve the Internet. The Applications Research Group (ARG) is an auxiliary team that aims to bolsters the mission by focusing on shipping tools that people love to use on the Internet. -- more at https://arg.protocol.ai
```

Comment:
I assume this part is forgotten to be updated since this project is initialized by neither PL nor ARG. Can you please fix it and use the right intro of your team.